R/init.R

```diff
@@ -138,7 +138,7 @@ build_site_meta <- function(pkg = ".") {
 }
 
 site_meta <- function(pkg) {
-  article_index <- set_names(path_file(pkg$vignettes$file_out), pkg$vignettes$name)
+  article_index <- set_names(pkg$vignettes$file_out, path_file(pkg$vignettes$name))
```

Comment:
I haven't tested this but saw that downlit would use it in the same way as the other one.

However, maybe that part is used by other things that might break?